ui/qutilities.py

```diff
@@ -560,8 +560,8 @@ def addClickHandler(wgt, cb):
     border-left-style:solid;
     background-color:transparent;
 }
-QComboBox::drop-down:hover {
-    background-color:#f1fff9;
+QComboBox QAbstractItemView {
+    selection-color: #33aa33;
```

Comment:
Feel free to fiddle with this green text color if you'd like. I didn't put a lot of thought into picking this color.

Reply:
I like green.